### PR TITLE
Publish mpTelemetry thirdparty packages in jar

### PR DESCRIPTION
**Pull request by Yas-Franco**
For issue #23080

### CODE REVIEW

**Comment by Yas-Franco** [comment]
#build

**Comment by Yas-Franco** [comment]
#build

**Comment by Yas-Franco** [comment]
#libby

File: dev/io.openliberty.io.opentelemetry.internal/bnd.bnd [hunk 1]
```diff
@@ -13,6 +13,8 @@
 bVersion=1.0
 openTelemetryVersion=1.19.0
 
+src: src
```
Comment (tevans78) [inline dev/io.openliberty.io.opentelemetry.internal/bnd.bnd:16]: In a follow on PR, I think this bundle is now just a rebundling of io.opentelemetry and not an internal integration. Therefore (being a bit picky) the name changes to just `io.openliberty.io.opentelemetry`.

**Review by benjamin-confino — DISMISSED** [review]

File: dev/io.openliberty.mpTelemetry.1.0.thirdparty/bnd.bnd [hunk 2]
```diff
@@ -11,19 +11,16 @@
...
 
-src: src
+openTelemetryVersion=1.19.0
+alphaOpenTelemetryVersion=1.19.0.alpha
 
-Export-Package: \
-  io.opentelemetry.api; version=${openTelemetryVersion},\
-  io.opentelemetry.api.baggage; version=${openTelemetryVersion},\
+Private-Package: \
   io.opentelemetry.api.common; version=${openTelemetryVersion},\
```
Comment (Azquelt) [inline dev/io.openliberty.mpTelemetry.1.0.thirdparty/bnd.bnd:26]: What happened to these two packages. Shouldn't they still be included?

**Review by Azquelt — CHANGES_REQUESTED** [review]
Looks like two packages are missing from the API jar?

**Comment by Yas-Franco** [comment]
#libby

File: dev/io.openliberty.mpTelemetry.1.0.thirdparty/bnd.bnd (hunk 2 above)
Comment (Yas-Franco) [reply dev/io.openliberty.mpTelemetry.1.0.thirdparty/bnd.bnd:26]: I have added them back in now

File: dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry-1.0.feature [hunk 3]
```diff
@@ -3,7 +3,7 @@ symbolicName=io.openliberty.org.eclipse.microprofile.telemetry-1.0
 visibility=private
 singleton=true
 -features=io.openliberty.mpCompatible-6.0
--bundles=io.openliberty.mpTelemetry.1.0.thirdparty; location:="dev/api/third-party/,lib/"
```
Comment (Azquelt) [inline dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry-1.0.feature:6]: I'm not sure why we list `dev/api/third-party/` and `lib/` here.

Does the jar actually end up in both locations?

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/telemetry/microprofile-config.properties [hunk 4]
```diff
@@ -1,4 +1,5 @@
 otel.exporter.jaeger.endpoint=http://localhost:14250
 otel.traces.exporter=jaeger
 otel.exporter.jaeger.timeout=10000
-otel.experimental.sdk.enabled=true
\ No newline at end of file
+otel.sdk.disabled=false
+otel.service.name=fat_test
```
Comment (Azquelt) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/telemetry/microprofile-config.properties]: Does this relate to this change? Looks like it was for something else?

Comment (Azquelt) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/telemetry/microprofile-config.properties]: This change doesn't seem to be related to the rest of the PR.

Was this just something which was forgotten in a previous PR?

**Review by Azquelt — COMMENTED** [review]
I've added a couple of questions.

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/telemetry/microprofile-config.properties (hunk 4 above)
Comment (Yas-Franco) [reply dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/telemetry/microprofile-config.properties]: Yes the FAT still used the old config property

File: dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry-1.0.feature (hunk 3 above)
Comment (Yas-Franco) [reply dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry-1.0.feature:6]: You're right. It doesn't get published to the lib folder anyway

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/telemetry/microprofile-config.properties (hunk 4 above)
Comment (Azquelt) [reply dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/telemetry/microprofile-config.properties]: ok

File: dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry-1.0.feature (hunk 3 above)
Comment (Azquelt) [reply dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry-1.0.feature:6]: Can fix this as a follow up

**Review by Azquelt — APPROVED** [review]
